odl/__init__.py

```diff
@@ -41,6 +42,8 @@
 from . import phantom
 from . import deform
 from . import ufunc_ops
+from . import datasets
```

Comment:
Wouldn't we run into the risk of said package becomming overpopulated? With that said I agree that there is no real sharp limit, e.g. where would we put complicated humanoid phantoms?

I'll need to think more about it.